content/en/tracing/setup_overview/open_standards/otel_collector_datadog_exporter.md

```diff
@@ -64,8 +64,9 @@ processors:
 exporters:
   datadog/api:
 
-    tags:
-      - example:tag
+    host_metadata:
+      tags:
+        - example:tag
 
     api:
       key: aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```

Comment:
Not that you change that and it is not a big deal at all but IMHO aaaaa... looks bit strange. Arguably xxxxxx... or XXXX... more traditional substitution. And I think [xxxxx...] or <xxx...> have a better implied intentions.

Reply:
👍 that's a good point. I think the right choice here is to put as an example:

```
api:
  key: ${DD_API_KEY}
```

and [pass the API key as an environment variable](https://opentelemetry.io/docs/collector/configuration/#configuration-environment-variables), which is the recommended setup for sensitive fields.

This needs to change in a few places though and I think I want to wait a bit until we have made a change on environment variable resolution, but I will make a note, thanks!

Reply:
@julien-lebot 
```
I've seen aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa used for API key at Datadog more than once, and I personally prefer it to xxxx or XXXX.
"a" is the first letter of the alphabet and it sounds like someone saying "aaaaaaah", like "aaaaah we're missing the API key".
But you can't pronounce "XXXXX" :)
I don't particularly agree that "XXXX" has better implied intentions.
````
That teaches me that what I am very biased even about most obvious (for me) thing :)